1.8 rc2 CUDA + Qt Compiler errors on Mac OS X

Relevant to 1.8 release issue #1373

I'm encountering quite a few compiler issues on OS X:

[Full make -i output](https://gist.github.com/ahundt/cfd6fca1499361cab27d)


[ 68%] Building NVCC (Device) object gpu/features/CMakeFiles/pcl_gpu_features.dir/src/pcl_gpu_features_generated_fpfh.cu.o
/Users/athundt/source/git/pcl/gpu/features/src/fpfh.cu(235): error: "__any" is ambiguous

1 error detected in the compilation of "/var/folders/9f/jr_p974d3j318tmvrfkjl55w0000gp/T//tmpxft_00008583_00000000-19_fpfh.compute_52.cpp1.ii".
CMake Error at pcl_gpu_features_generated_fpfh.cu.o.cmake:266 (message):
  Error generating file
  /Users/athundt/source/git/pcl/build/gpu/features/CMakeFiles/pcl_gpu_features.dir/src/./pcl_gpu_features_generated_fpfh.cu.o


CUDA errors:
```
[ 68%] Building NVCC (Device) object gpu/features/CMakeFiles/pcl_gpu_features.dir/src/pcl_gpu_features_generated_fpfh.cu.o
/Users/athundt/source/git/pcl/gpu/features/src/fpfh.cu(235): error: "__any" is ambiguous

1 error detected in the compilation of "/var/folders/9f/jr_p974d3j318tmvrfkjl55w0000gp/T//tmpxft_00008583_00000000-19_fpfh.compute_52.cpp1.ii".
CMake Error at pcl_gpu_features_generated_fpfh.cu.o.cmake:266 (message):
  Error generating file
  /Users/athundt/source/git/pcl/build/gpu/features/CMakeFiles/pcl_gpu_features.dir/src/./pcl_gpu_features_generated_fpfh.cu.o


[ 68%] Building NVCC (Device) object gpu/features/CMakeFiles/pcl_gpu_features.dir/src/pcl_gpu_features_generated_fpfh.cu.o
/Users/athundt/source/git/pcl/gpu/features/src/fpfh.cu(235): error: "__any" is ambiguous

1 error detected in the compilation of "/var/folders/9f/jr_p974d3j318tmvrfkjl55w0000gp/T//tmpxft_000085b0_00000000-19_fpfh.compute_52.cpp1.ii".
CMake Error at pcl_gpu_features_generated_fpfh.cu.o.cmake:266 (message):
  Error generating file
  /Users/athundt/source/git/pcl/build/gpu/features/CMakeFiles/pcl_gpu_features.dir/src/./pcl_gpu_features_generated_fpfh.cu.o


[ 68%] Linking CXX shared library ../../lib/libpcl_gpu_features.dylib
clang: error: no such file or directory: 'CMakeFiles/pcl_gpu_features.dir/src/pcl_gpu_features_generated_fpfh.cu.o'
```

The Qt errors are much more numerous, it probably makes sense to just check the link above for those.

--- a/gpu/features/src/fpfh.cu
+++ b/gpu/features/src/fpfh.cu
@@ -47,7 +47,6 @@
 
 using namespace pcl::gpu;
 using namespace pcl::device;
-using namespace std;
 
 #ifndef M_PI
 #define M_PI 3.14159265358979323846f

